docs/docs/command-line-interface.mdx

```diff
@@ -31,6 +31,12 @@ import RasaProBanner from "@theme/RasaProBanner";
 |`rasa evaluate markers` |Extracts markers from an existing tracker store.                                                                                          |
 |`rasa -h`               |Shows all available commands.                                                                                                             |
 
+:::note
+If you run into character encoding issues on Windows like: `UnicodeEncodeError: 'charmap' codec can't encode character ...` or
+the terminal is not displaying colored messages properly, prepend `winpty` to the command you would like to run.
```

Comment:
To address these Typo CI warnings, you can add `winpty` to this list [here](https://github.com/RasaHQ/rasa/blob/44741a7b014e8d53608e069b84b43a33e2925383/.typo-ci.yml#L64).

Reply:
Added 🙏